salt/docker_apps/files/kingfisher_process.yaml

```diff
@@ -13,7 +14,9 @@ x-daemon: &daemon
 services:
   django:
     <<: *daemon
-    command: "./gunicorn_runner.sh {{ pillar.docker_apps.kingfisher_process.port }}"
+    # The container port must match the one exposed in the Dockerfile
```

Comment:
This comment is obvious, no? (Here and elsewhere)

Reply:
Removed